lib/migrate/MigrationGenerator.js

```diff
@@ -35,11 +36,24 @@ class MigrationGenerator {
     return Promise.all(promises);
   }
 
-  _getStubPath() {
-    return (
-      this.config.stub ||
-      path.join(__dirname, 'stub', this.config.extension + '.stub')
-    );
+  async _getStubPath() {
```

Comment:
is this completely identical to what Seeder is doing? can this be extracted to some util?

Reply:
It is, I think the code was duplicated in the current master as well. This PR was already getting large, so It seemed like a good idea not to make *too* many changes; that way the relationship with the current version could be a bit clearer for any reviewers/maintainers

Reply:
fair